xml/art_solution_monitoring.xml

```diff
@@ -0,0 +1,249 @@
+    <para>
+      Export &hana; database metrics.
+      The exporter is able to export the
+      metrics from more than one database or tenant. This option
+      (<option>multi_tenant</option>) is enabled by default.
+    </para>
+    <formalpara>
+      <title>Package:</title>
+      <para><package>prometheus-hanadb_exporter</package></para>
+    </formalpara>
+    <formalpara>
+      <title>Link:</title>
+      <para>
+        See <link xlink:href="https://github.com/SUSE/hanadb_exporter"/>.
+      </para>
+    </formalpara>
+    <itemizedlist>
+      <title>Export Metrics in Prometheus Format</title>
+      <listitem>
+        <para>
+          Memory metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          CPU metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Disk usage metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          I/O metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Network metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Top queries consuming time and memory
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          System replication metrics
+        </para>
+      </listitem>
+    </itemizedlist>
+    <remark>toms 2020-07-02: Regarding Installation: Do we need really need to
+      describe this via a virtual Python environment or can this be installed
+      through an RPM package?</remark>
+    <!--
+    1. Plain text password
+    2. HANA Keystore
+    3. SAP HANA Driver installation.
+    -->
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-ha-cluster-exp">
+    <title>&ha; Cluster Exporter</title>
+    <para>
+      Enables monitoring of Pacemaker on &ha; clusters.
+      Collects metrics to easily monitor cluster status and health.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>
+        See <link xlink:href="https://github.com/ClusterLabs/ha_cluster_exporter"/>.
+      </para>
+    </formalpara>
+    <itemizedlist>
+      <title>Export Metrics in Prometheus Format</title>
+      <listitem>
+        <para>
+          Pacemaker cluster summary, nodes, and resource status
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Corosync ring errors and quorum votes.
+          Currently, only Corosync version&nbsp;2 is supported.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          SBD devices health status
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          DRBD resources and connections status.
+          Currently, only DRBD version&nbsp;9 is supported.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-sap-host-exp">
+    <title>&sap; Host Exporter</title>
+    <para>
+      Enables the monitoring of &netweaver; applications.
```

Comment:
This is partially true as well. This exporter gets metrics from the SAP host process. This process is available in other instances like HANA, S4HANA instances, etc. It doesn't only belong to Netweaver

Reply:
We could actually complement this saying that the gathered metrics are the data that can be obtained running `sapcontrol` command, that is the interface that most of the SAP admins use

Reply:
Ahh, good. Ok, I've rephrased the sentence:

    Enables the monitoring of &netweaver;, &hana;, and other applications.
    The gathered metrics are the data that can be obtained by running the
     <command>sapcontrol</command> command.